core/generator.js

```diff
@@ -158,11 +158,12 @@ Blockly.Generator.prototype.allNestedComments = function(block) {
 /**
  * Generate code for the specified block (and attached blocks).
  * @param {Blockly.Block} block The block to generate code for.
+ * @param {boolean} opt_thisOnly True to generate code for only this block.
  * @return {string|!Array} For statement blocks, the generated code.
  *     For value blocks, an array containing the generated code and an
  *     operator order value.  Returns '' if block is null.
  */
-Blockly.Generator.prototype.blockToCode = function(block) {
+Blockly.Generator.prototype.blockToCode = function(block, opt_thisOnly) {
   if (!block) {
     return '';
   }
```

Comment:
What happens if a statement block is disabled and blockToCode is called on it with opt_thisOnly set true?  I'd expect empty string, but my reading of the code indicates that the entire remainder of the stack is returned.